docs/general/dql/selects.rst

```diff
@@ -485,6 +487,14 @@ does always return ``NULL`` when comparing ``NULL``.
     +----------+
     SELECT 1 row in set (... sec)
 
+.. CAUTION::
+
+    ``IS NULL`` clauses can slow a query down because CrateDB has to iterate
+    over all rows for the comparison and cannot utilize the index.
```

Comment:
This is not accurate. In most cases `IS NULL` can utilize an index. Cases where it can't would be if it is on a nested column of an ignored object field.